src/Bicep.LangServer.UnitTests/BicepExternalSourceRequestHandlerTests.cs

```diff
@@ -289,7 +285,7 @@ public async Task RestoredValidModule_WithSource_RequestingCompiledJson_ShouldRe
 
             var bicepSource = "metadata hi = 'This is the bicep source file'";
             var bicepUri = new Uri("file:///foo/bar/entrypoint.bicep");
-            var sourceArchive = SourceArchive.UnpackFromStream(SourceArchive.PackSourcesIntoStream(bicepUri, new Core.Workspaces.ISourceFile[] {
+            var sourceArchive = SourceArchive.UnpackFromStream(SourceArchive.PackSourcesIntoStream(bicepUri, null, new Core.Workspaces.ISourceFile[] {
```

Comment:
- Similar to other comments:  I recommend qualifying `documentLinks: null` to improve readability
- The qualifier Core.Workspaces here is not necessary and can be simplified